pkgpanda/docker/dcos-builder/Dockerfile

```diff
@@ -37,12 +37,18 @@ RUN apt-get -qq update && apt-get -y install \
   python-setuptools \
   ruby \
   scala \
+  software-properties-common \
```

Comment:
While you're at it, can you also remove the duplicate `libc6-dev` on line 18 (properly alphabetized on line 20)?